css/custom.css

```diff
@@ -184,14 +190,15 @@ form > textarea {
   -webkit-box-align: center;
       -ms-flex-align: center;
           align-items: center;
+  height: 75px;
 }
 
 .socials > a {
```

Comment:
reiktų scss nuo šitos vietos į .socials sumesti, nes perrašai container>bg-primary klasę, o prie 'wt' ir 'padding' klassių gali prieiti betkas, ir gali pradėti pjautis css taisyklės jei kas pradės šitą klasę taikyti perrašys blogai.

Reply:
Sutvarkiau, manau